ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingManager.java

```diff
@@ -0,0 +1,66 @@
+package us.ihmc.behaviors.activeMapping;
+
+import us.ihmc.avatar.drcRobot.DRCRobotModel;
+import us.ihmc.avatar.drcRobot.ROS2SyncedRobotModel;
+import us.ihmc.avatar.ros2.ROS2ControllerHelper;
+import us.ihmc.communication.property.ROS2StoredPropertySetGroup;
+import us.ihmc.footstepPlanning.communication.ContinuousWalkingAPI;
+import us.ihmc.perception.gpuHeightMap.RapidHeightMapExtractor;
+import us.ihmc.ros2.ROS2Node;
+import us.ihmc.sensorProcessing.heightMap.HeightMapData;
+import us.ihmc.tools.thread.RestartableThrottledThread;
+
+/**
+ * This class takes care of stuff to run continuous hiking in the {@link us.ihmc.PerceptionAndAutonomyProcess}.
```

Comment:
If this class is specific to PerceptionAndAutonomyProcess, it should go in the same package as it.